applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h

```diff
@@ -174,7 +174,7 @@ namespace Kratos
 							if (Length < mElemSize) mElemSize = Length;
 						}
 					mElemSize = sqrt(mElemSize);
-					ielem->GetValue(MEAN_SIZE) = mElemSize;
+					ielem->SetValue(MEAN_SIZE,mElemSize);
```

Comment:
Why changing to the `SetValue`? Isn't it initialized before?

Reply:
No. And when I run it in FullDebug it crashed...

Reply:
Well, I see now, it is initialized before... I don't know why it crashed in FullDebug then...

Reply:
I see it... The variables are initialized as **nodal non-historical variables**, but they are being used **elementary**...

Reply:
I just corrected the initialization